/*
  Vehicle Data

  This API section holds messages related to OTA Vehicle Data between device and backend.

  The version of the OpenAPI document: 3

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.vehicledata.v3;

import public "backend_to_device_message_meta.proto";

message DeactivateMeasurementCommand {

  BackendToDeviceMessageMeta meta = 3347973;

  // The backend-side measurementConfigurationId. The measurement which should be deactivated. In case of the AA-stack this identifier is contained in ZIP-file which is sent to the device with activateMeasurementCommand.
  string measurementIdentifier = 525670011;

}
